packages/babel-register/src/node.js

```diff
@@ -129,20 +111,27 @@ function hookExtensions(_exts) {
   });
 }
 
-hookExtensions(util.canCompile.EXTENSIONS);
+hookExtensions([".js", ".jsx", ".es6", ".es"]);
 
 export default function (opts?: Object = {}) {
-  if (opts.only != null) only = util.arrayify(opts.only, util.regexify);
-  if (opts.ignore != null) ignore = util.arrayify(opts.ignore, util.regexify);
-
-  if (opts.extensions) hookExtensions(util.arrayify(opts.extensions));
+  if (opts.extensions) hookExtensions(opts.extensions);
 
   if (opts.cache === false) cache = null;
 
   delete opts.extensions;
-  delete opts.ignore;
   delete opts.cache;
```

Comment:
These deletes are removing options that are specific to `babel-register`, so this is deleting it because `babel-core` will throw on the unknown option.